docs/user/aws/limits.md

```diff
@@ -23,20 +23,22 @@ limit.
 
 ## Elastic Network Interfaces (ENI)
 
-The default installation creates 21 + the number of availability zones of ENIs (e.g. us-east-1 = 21 + 6 = 27 ENIs).
+The default installation creates 21 + the number of availability zones of ENIs (e.g. 21 + 3 = 24 ENIs for a three-zone cluster).
 The default limit per region is 350. Additional ENIs are created for additional machines and elastic load balancers
 created by cluster usage and deployed workloads. A service limit increase here may be required to satisfy the needs of
 additional clusters and deployed workloads.
 
 ## Elastic IP (EIP)
 
-For a single, default cluster, your account will have the needed capacity limits required. There is one exception,
-"EC2-VPC Elastic IPs". The installer creates a public and private subnet for each
-[availability zone within a region][availability-zones] to provision the cluster in a highly available configuration. In
-each private subnet, a separate [NAT Gateway][nat-gateways] is created and requires a separate [elastic IP][elastic-ip].
-The default limit of 5 is sufficient for most regions and a single cluster. For the us-east-1 region, a higher limit is
-required. For multiple clusters, a higher limit is required. Please see [this map][az-map] for a current region map with
-availability zone count. We recommend selecting regions with 3 or more availability zones.
+By default, the installer distributes control-plane and compute machines across [availability zone within a region][availability-zones] to provision the cluster in a highly available configuration. 
```

Comment:
nit: `all availability zones`

Reply:
> nit: all availability zones

But it's just "as many of them as we need to fill the requested `replicas` for each pool", right?  I'm fine adding "all" if you want, but I don't want folks thinking that the default for a six-zone region will be creating resources in all six zones.  You still want the "all"?

Reply:
`By default,` means default configuration, right? that means all azs in the region.. Am i missing something?

Reply:
> `By default`, means default configuration, right? that means all azs in the region.. Am i missing something?

By default we use as many of the available (#1210) availablity zones as we need to hold the default three nodes (of each node type) while distributing each type as much as possible.  So while all available zones are on the table (with the default logic), we're only ever going to distribute machines over the first three zones (with the default replica counts).

Reply:
> > `By default`, means default configuration, right? that means all azs in the region.. Am i missing something?
> 
> By default we use as many of the available (#1210) availablity zones as we need to hold the default three nodes (of each node type) while distributing each type as much as possible. So while all available zones are on the table (with the default logic), we're only ever going to distribute machines over the first three zones (with the default replica counts).

sure, anyway it should be `availability zone*s*`

Reply:
> ... anyway it should be availability zone*s*

Fixed in 009963e27 -> ba76eea1f.